scripts/uninstall-zowe.sh

```diff
@@ -211,10 +211,10 @@ echo
 PROCLIBS=$("${CIZT_INSTALL_DIR}/opercmd" '$d proclib' | grep 'DSNAME=.*\.PROCLIB' | sed 's/.*DSNAME=\(.*\)\.PROCLIB.*/\1.PROCLIB/')
 
 ################################################################################
-# removing ZOWESVR
+# removing ZWESVSTC
```

Comment:
It's better to also delete old ZOWESVR, ZWESIS01, ZWESAUX if they exist. But it's just for cleaning up purpose, shouldn't affect new instance of Zowe startup.